### PR TITLE
adds ascension achievement

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds an ascension achievement for ascending His Grace

## Why It's Good For The Game

some permanent bragging rights as a reward if you actually manage to do this

## Changelog
:cl:
add: Caedite eos. Novit enim Dominus qui sunt eius.
/:cl:

### CODE REVIEW

File: code/datums/achievements/misc_achievements.dm [hunk 1]
```diff
@@ -111,3 +111,9 @@
 	name = "HA, GOTTEM"
 	desc = "Made you look!"
 	database_id = MEDAL_GOTTEM
+
+/datum/award/achievement/misc/ascension
+	name = "Ascension"
```
Comment (ATH1909) [inline code/datums/achievements/misc_achievements.dm:116]: might want to make it clear that this refers to a His Grace ascension, not a heretic one.

Comment (Fikou) [reply code/datums/achievements/misc_achievements.dm:116]: its obvious

**Comment by tralezab** [comment]
the achievement icon needs to be added to the asset spritesheet

File: code/game/objects/items/his_grace.dm [hunk 2]
```diff
@@ -252,3 +252,4 @@
 	if(istype(master))
 		master.visible_message("<span class='his_grace big bold'>Gods will be watching.</span>")
 		name = "[master]'s mythical toolbox of three powers"
+		master.client.give_award(/datum/award/achievement/misc/ascension, master)
```
Comment (Rohesie) [inline code/game/objects/items/his_grace.dm:255]:
```suggestion
		master.client?.give_award(/datum/award/achievement/misc/ascension, master)
```

**Comment by AnturK** [comment]
The item is bad enough, i don't want to encourage contextless murderbone through achievements.

**Comment by Fikou** [comment]
help

**Pull request merged**